### DISCUSSION TITLE
Unix compilation

### DISCUSSION THREAD

**Discussion by lurpi** [Anura3D/Anura3D_OpenSource#234] [General]
Dear Anura3D devs,

it is a cool code, thanks for the nice workshop past week. I am trying to compile under Linux, but I struggle just because there is no clear order of the various .FOR files. 
Would you be able to provide a cmake file or similar, additionally to the Visual Studio solution file? 
Maybe even a log from Visual Studio compilation is enough, to get the right order of compilation.

Thanks!

**Comment by pmarveggio** [comment]
@lurpi Hi! We will shortly release a cmake file, we are testing before releasing. I will let you know here when it will be available

**Comment by zorroxt1118** [comment]
Hello, May I kindly inquire if there is an update for the cmake file?
